modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/form_fields.rb

```diff
@@ -44,18 +44,6 @@ def insurance_policy_number
           'form1[0].#subform[2].ClaimantsLastName[1]'
         end
 
-        def mailing_address_street
```

Comment:
Were these methods removed intentionally? What replaces them if so?

Reply:
They're stamped and auto-sized fields. We don't call these methods...they were just hanging around.